pkgs/tools/misc/fontforge/default.nix

```diff
@@ -10,6 +10,8 @@
 stdenv.mkDerivation rec {
   name = "fontforge-${version}";
   version = "20160404";
+  versionModtime = "1459728000"; # unix timestamp of ${version}
```

Comment:
Nix 1.12 has an `exec` primop which I believe is intended to supplant most simple uses of IFD.

(What I had in mind here, however, was to simply compute the derived values before use in `postPatch`)